api/tests/AppBundle/Service/OrgServiceTest.php

```diff
@@ -369,9 +369,9 @@ public function testPAAddFromCasrecRows()
         $this->pa->addFromCasrecRows($dataMove);
         self::$em->clear();
 
-        // check client 3 is now associated with deputy1
-        $this->assertCount(1, self::$fixtures->findNamedDeputyByNumber('00000001')->getClients());
-        $this->assertCount(2, self::$fixtures->findNamedDeputyByNumber('00000002')->getClients());
+        // check client 2 remains with deputy 1
+        $this->assertCount(2, self::$fixtures->findNamedDeputyByNumber('00000001')->getClients());
+        $this->assertCount(1, self::$fixtures->findNamedDeputyByNumber('00000002')->getClients());
         $this->assertCount(1, self::$fixtures->findNamedDeputyByNumber('00000003')->getClients());
 
```

Comment:
Out of scope for this ticket but it would be nice to re-write this into a few individual tests as its not clear what's covered at first glance.

Reply:
Completely agree. The `OrgService` is not in a position for a reasonably straight forward rewrite and a much needed rewrite of the tests